mlir_graphblas/src/lib/GraphBLAS/GraphBLASUtils.cpp

```diff
@@ -0,0 +1,112 @@
+#include "mlir/IR/Attributes.h"
+#include "mlir/IR/BuiltinOps.h"
+#include "mlir/IR/BuiltinTypes.h"
+#include "mlir/Dialect/StandardOps/IR/Ops.h"
+#include "mlir/Dialect/SparseTensor/IR/SparseTensor.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
+
+using namespace std;
+using namespace mlir;
+using namespace mlir::sparse_tensor;
+
+// make CSR tensor type
+mlir::RankedTensorType getCSRTensorType(MLIRContext *context, Type valueType)
+{
+    SmallVector<sparse_tensor::SparseTensorEncodingAttr::DimLevelType, 2> dlt;
+    dlt.push_back(sparse_tensor::SparseTensorEncodingAttr::DimLevelType::Dense);
+    dlt.push_back(sparse_tensor::SparseTensorEncodingAttr::DimLevelType::Compressed);
+    unsigned ptr = 64;
+    unsigned ind = 64;
+    AffineMap map = AffineMap::getMultiDimIdentityMap(2, context);
+
+    RankedTensorType csrTensor = RankedTensorType::get(
+        {-1, -1}, /* 2D, unknown size */
+        valueType,
+        sparse_tensor::SparseTensorEncodingAttr::get(context, dlt, map, ptr, ind));
+
+    return csrTensor;
+}
+
+/// Returns function reference (first hit also inserts into module).
+// from: llvm/llvm-project/mlir/lib/Dialect/SparseTensor/Transforms/SparseTensorConversion.cpp
+static FlatSymbolRefAttr getFunc(mlir::ModuleOp &mod, mlir::Location &loc, StringRef name, TypeRange result,
```

Comment:
I think we can remove the `mlir::` prefixes because of `using namespace mlir;` above. It would help the code look more consistent since we in other places use `MLIRContext` instead of `mlir::MLIRContext`.

I think we can do this throughout this file.

Reply:
I will make this change as well.